### PR TITLE
Updating learning rate with special warm up in examples

### LINKED ISSUE #566: Bug in run_classifier.py fp16 learning rate
After the latest update, my learning rate of fp16 in run_classifier.py keeps increasing.

https://github.com/huggingface/pytorch-pretrained-BERT/blob/2dee86319dbad575352358b8f2fb4129940e381a/examples/run_classifier.py#L857-L858

I think the right code is: lr_this_step = args.learning_rate * warmup_linear.get_lr(global_step, args.warmup_proportion). 

https://github.com/huggingface/pytorch-pretrained-BERT/blob/2dee86319dbad575352358b8f2fb4129940e381a/pytorch_pretrained_bert/optimization.py#L53-L62
In this function, the first argument should be step, and global_step/num_train_optimization_steps makes process calculated again and when input process to WarmupLinearSchedule, it would be too small to decrease
https://github.com/huggingface/pytorch-pretrained-BERT/blob/2dee86319dbad575352358b8f2fb4129940e381a/pytorch_pretrained_bert/optimization.py#L162-L171

### PR DESCRIPTION
**Pull request by burcturkoglu**
Updating examples by removing division to num_train_optimization_steps for new WarmupLinearSchedule.

Fixes #566

### CODE REVIEW

**Comment by thomwolf** [comment]
Oh great thanks Burc!
